modules/apps/29-fee/ibc_middleware_test.go

```diff
@@ -135,7 +138,7 @@ func (suite *FeeTestSuite) TestOnChanOpenInit() {
 
 					suite.Require().NoError(err, "unexpected error from version: %s", tc.version)
 				} else {
-					suite.Require().Error(err, "error not returned for version: %s", tc.version)
+					suite.Require().True(errors.Is(err, tc.expErr) || strings.Contains(err.Error(), tc.expErr.Error()), "error no returned for version: %s or error %s is not %s", tc.version, err, tc.expErr)
```

Comment:
This line will come up several times in the PR. It's ugly, but the issue is that some of the errors that we get (especially from `sdk`) are not wrapped, so we can't check for a specific error but rather we have to rely on the error string. It's not ideal, but the other option would be to catch every usage of those errors and wrap them ourselves with something else